modules/configuring-firewall.adoc

```diff
@@ -102,7 +102,7 @@ You can use the wildcards `\*.quay.io` and `*.openshiftapps.com` instead of `cdn
 .16+|AWS
 |`aws.amazon.com`
 |443
-|Used to install and manage clusters in an AWS environment.
+|Used to install and manage clusters in an AWS environment. If you use the AWS Security Token Service (STS) with the Cloud Credential Operator (CCO), you must provide access to CloudFront.
```

Comment:
IIUC, CloudFront is required only if the private S3 bucket is used. 
@jianping-shu can you help confirm?

Reply:
Yes. I think the document update should be like below:
URL: *.cloudfront.net       
Function: If use AWS STS and the private S3 bucket...
Refer to this [comment](https://issues.redhat.com/browse/OCPBUGS-27214?focusedId=24063819&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-24063819) for the background

Reply:
@jianping-shu, Thanks for your reply. Could you review these updates to this page: 
1) Added this section to AWS section of table in step 4: "If you use AWS Security Token Service (STS) and the private S3 bucket, you must provide access to CloudFront." 
2) Added new entry to 'Additional resources': "For more information about providing access to CloudFront if you use AWS STS and the private S3 bucket, see OpenID Connect requirements for AWS STS."

Reply:
@brendan-daly-red-hat Thanks for the update! IMO it is better to list the URL: *.cloudfront.net in the firewall table explicitly.

Reply:
@jianping-shu, Thanks for the reviewing the update. Unfortunately, we can't add `xrefs` within modules. This means that the link can only be added to the 'Additional resources' section.

Reply:
@brendan-daly-red-hat I am OK with the 'Additional resources'. I was suggesting to add one more row for *.cloudfront.net in the firewall table so it won't be overlooked by the user.